### PR TITLE
fix(FFI): only use native types u8, i8, u32, i32, etc.

### PR DESCRIPTION
**Pull request by Manuthor**

### CODE REVIEW

File: crates/aesgcm/Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof_aesgcm"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (tbrezot) [inline crates/aesgcm/Cargo.toml:3]: Isn't this a `0.1.1`?

File: crates/anonymization/Cargo.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof_anonymization"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (tbrezot) [inline crates/anonymization/Cargo.toml:3]: It seems there is no change in this subcrate.

File: crates/cloudproof/Cargo.toml [hunk 3]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof"
-version = "2.1.0"
+version = "2.1.1"
```
Comment (tbrezot) [inline crates/cloudproof/Cargo.toml:3]: Same here

File: crates/ecies/Cargo.toml [hunk 4]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "cloudproof_ecies"
-version = "0.1.0"
+version = "0.2.0"
```
Comment (tbrezot) [inline crates/ecies/Cargo.toml:3]: why isn't it a patch release?

File: crates/fpe/Cargo.toml [hunk 5]
```diff
@@ -36,7 +36,7 @@ pyo3 = { workspace = true, features = ["extension-module"], optional = true }
 wasm-bindgen = { workspace = true, optional = true }
 
 [dev-dependencies]
-criterion = { version = "0.4.0", default-features = false }
+criterion = { version = "0.5.1", default-features = false }
```
Comment (tbrezot) [inline crates/fpe/Cargo.toml:39]: Is this version bump on purpose?

**Review by tbrezot — COMMENTED** [review]
LGTM

Some questions about versionning only

File: crates/anonymization/Cargo.toml (hunk 2 above)
Comment (Manuthor) [reply crates/anonymization/Cargo.toml:3]: the underlying version of crypto_core is still 0.7.x

File: crates/aesgcm/Cargo.toml (hunk 1 above)
Comment (Manuthor) [reply crates/aesgcm/Cargo.toml:3]: right

File: crates/cloudproof/Cargo.toml (hunk 3 above)
Comment (Manuthor) [reply crates/cloudproof/Cargo.toml:3]: in the next release, we need to publish the new cloudproof crate (with refreshed deps)

File: crates/ecies/Cargo.toml (hunk 4 above)
Comment (Manuthor) [reply crates/ecies/Cargo.toml:3]: indeed

File: crates/fpe/Cargo.toml (hunk 5 above)
Comment (Manuthor) [reply crates/fpe/Cargo.toml:39]: reverted

**Pull request merged**